### PR TITLE
ClickTP Fix

### PR DESCRIPTION
**Pull request by weedelbhoy1**

### CODE REVIEW

File: Utils/Target.cpp [hunk 1]
```diff
@@ -1,85 +1,85 @@
...
+	static auto hitboxMod = moduleMgr->getModule<Hitbox>();
+	static auto teams = moduleMgr->getModule<Teams>();
+	static auto noFriends = moduleMgr->getModule<NoFriends>();
+
+	if (!ent->isAlive())
+		return false;
+
+	auto entityTypeId = ent->getEntityTypeId();
+
+	if (antibot->isEntityIdCheckEnabled() && entityTypeId <= 130 && entityTypeId != 63)
```
Comment (weedelbhoy1) [inline Utils/Target.cpp:32]: fix is at line 32 idk why it did this github being a retardatron i guess

Comment (N0tExisting) [reply Utils/Target.cpp:32]: Did you change the indentation?

**Comment by Intoprelised** [comment]
im just gonna assume this works

**Comment by GodGamer029** [comment]
Fix your git pls

**Pull request closed without merging**